node/cli/src/chain_spec.rs

```diff
@@ -152,6 +152,12 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 			burn: Permill::from_percent(50),
 		}),
 		contract: Some(ContractConfig {
+			signed_claim_handicap: 2,
```

Comment:
The common configuration here should be factored out.

Reply:
I can do it but none of the other config does it.

This could be made in a specific another that changes all of them.